cypress/plugins/index.js

```diff
@@ -9,7 +9,7 @@ export default (on, config) => {
 
   config.env.basic_username = process.env.BASIC_AUTH_USERNAME
   config.env.basic_password = process.env.BASIC_AUTH_PASSWORD
-  config.env.baseUrl = process.env.BASE_URL
+  config.env.basic_url = process.env.BASIC_URL
```

Comment:
hm :)

Reply:
what's wrong here?

Reply:
I just didn't understand why you change it back. We change it from "basic" to "base" in the previous PR.